Cargo.toml

```diff
@@ -70,100 +70,100 @@ wiremock = { version = "0.5", default-features = false }
-sp-core = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-core-hashing-proc-macro = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-inherents = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-io = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-keyring = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-keystore = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-offchain = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-panic-handler = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-runtime = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-session = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-staking = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-std = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-timestamp = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-tracing = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-transaction-pool = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-sp-version = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-substrate-frame-rpc-system = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-substrate-wasm-builder = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
-try-runtime-cli = { git = "https://github.com/humanode-network/substrate", tag = "locked/polkadot-v0.9.40-2024-07-30", default-features = false }
+frame-benchmarking = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.40-1", default-features = false }
```

Comment:
Let's formally define the rules we follow when naming the branches, create the normative documentation for this and start strictly following the procedure

Reply:
Absolutely agree! Will follow with main branch (`h-polkadot-v*`) in forks and tag if we need to take particular code.

Reply:
https://github.com/humanode-network/humanode/pull/1117/commits/ef9be357ea632fe78c1d9948e6a8ee75692722c0